modal/image.py

```diff
@@ -677,10 +679,10 @@ def from_dockerhub(tag: str, setup_commands: list[str] = [], **kwargs) -> "_Imag
         )
 
     @staticmethod
-    def from_dockerfile(path: Union[str, Path]) -> "_Image":
+    def from_dockerfile(path: Union[str, Path], context_mount: Optional[_Mount] = None) -> "_Image":
```

Comment:
Worth commenting the `context_mount` param imo. It combines Docker build context and Modal mount concepts, both of which are non-trivial concepts.

Reply:
For sure, adding comment